items.py

```diff
@@ -19,6 +17,24 @@ def sort_pubkey(key):
         }
     }
 
+svc_systemd = {
+    'ssh': {
+        'enabled': True,
+        'running': True,
+    }
+}
+
+sign_host_keys = {}
+if node.metadata.get('openssh').get('sign_host_keys').get('enabled'):
```

Comment:
this will break (exception), if the sign_host_keys key is not set. so i assume not intended

Reply:
Yep, but we set this in metadata.py defaults.